app/src/forms/form/exportService.js

```diff
@@ -133,7 +133,29 @@ const service = {
     };
   },
 
+  _readLatestFormSchema: async (formId) => {
```

Comment:
can remove async since there is no await for this function

Reply:
Could we also remove the async call on the function above called _formatSubmissionsJson?

Reply:
good catch. i've removed it from the _formatSubmissionsJson function.